README.md

```diff
@@ -52,7 +52,7 @@ Then consider a small [donation](https://opencollective.com/SuiteCRM/contribute)
 ### Security ###
 
 We take Security seriously here at SuiteCRM so if you have discovered a security risk report it by
-emailing [security@suitecrm.com](mailto:security@suitecrm.com). This will be delivered to the product team who handle security issues.
+emailing at [security@suitecrm.com](mailto:security@suitecrm.com). This will be delivered to the product team who handle security issues.
```

Comment:
This change doesn't make sense to me.

:+1: report it by emailing this@this.com
:-1: report it by emailing at this@this.com